Clean up language (#23432)
prestodb/presto: presto-docs/src/main/sphinx/installation/deployment.rst

    discovery-server.enabled=true
    discovery.uri=http://example.net:8080

> If a single coordinator is not sufficient, disaggregated coordinator setup can be used which supports multiple coordinators using the following minimal configuration:

Reviewer: ```suggestion
If a single coordinator is not sufficient, use a disaggregated coordinator setup which supports multiple coordinators using the following minimal configuration:
```
Nit to add "a" here as well, suggest change to active voice.

Author: done
